content/millennium/r4/financial/coverage.md

```diff
@@ -53,6 +53,7 @@ Search for Patient-level or Encounter-level Coverages that meet supplied query p
 _Implementation Notes_
   
 * Public Healthcare represents an insurance policy funded by a public health system such as a provincial or national health plan.
+* For Coverage search by -encounter, both `Encounter.read` and `Coverage.read` scopes are required
```

Comment:
nit: can we wrap '-encounter' in back tics too since it's a search param, i.e. `-encounter`